query-string/index.d.ts

```diff
@@ -13,7 +13,7 @@ declare module "query-string" {
      * Leading ? or # are ignored, so you can pass location.search or location.hash directly.
      * @param str
      */
-    export function parse(str: string): { [key: string]: string | string[] };
+    export function parse(str: string): { [key: string]: string | string[] | null };
```

Comment:
See the [falsy values section](https://github.com/sindresorhus/query-string#falsy-values).

```js
queryString.stringify({foo: undefined});
//=> ''
```

Not sure but it might be that it's best practice to turn on `strictNullChecks`. If that is the case, I'm definitely okay with that if we also add `| undefined`.